src/DDTrace/Http/Urls.php

```diff
@@ -2,12 +2,37 @@
 
 namespace DDTrace\Http;
 
-
 /**
  * A utility class that provides methods to work on urls
  */
 class Urls
 {
+    private static $defaultPatterns = [
+        // UUID's
+        '|\b([0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89AB][0-9a-f]{3}-[0-9a-f]{12})\b|i',
+        // 16-512 bit hex hashes
+        '|\b([0-9a-f]{4,128})\b|i',
```

Comment:
To be honest I would not add this. Isn't it too risky to have false positives with a 4-128 range? WDYT?

Reply:
From a security standpoint, I'd rather err on the side of over-obfuscation, but I totally see your point as well.

I was thinking about valid words that are 4+ characters that contain only a-f (like *cafe*). One way around erroneous obfuscation would be curating a whitelist of valid words that are 4+ and a-f. Perhaps we could eventually offer a feature to add words to the whitelist in the event there is a proper noun or a made-up word that's important (like *affeca*). There are also i18n considerations to support other languages.

In the end, a configurable whitelist that's also very fast (maybe making use of a Trie data structure or something) would be a bit of an undertaking, but it might be worth the tradeoff to ensure better security.

If you're OK with it, I'd like to ping someone from the security department to weigh in on this one. :)

Reply:
For now I've narrowed the scope of this to start at 32-bit hashes since an 8-character word with only a-f is much less likely to occur. :)